### PR TITLE
Optimize canvas loading

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1044 https://github.com/webstudio-is/webstudio-builder/issues/360

Canvas no longer need to load any data. No longer need pages. No longer need auth token. No longer need static css.

Here I cleaned all unnecessary logic to load canvas. It may load much faster now.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - test it on preview
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/routes/builder/$projectId.tsx [hunk 1]
```diff
@@ -45,15 +46,21 @@ export const loader = async ({
   const devBuild = await loadBuildByProjectId(project.id, "dev");
   const assets = await loadByProject(project.id, context);
 
-  const authReadToken = await createAuthReadToken({ projectId: project.id });
+  const end = Date.now();
+
+  const diff = end - start;
+
+  // we need to log timings to figure out how to speend up loading
```
Comment (istarkov) [inline apps/builder/app/routes/builder/$projectId.tsx:53]: speed up

**Review by istarkov — APPROVED** [review]

**Review by kof — APPROVED** [review]
is there anything we also need to do in saas for this?

**Comment by istarkov** [comment]
https://webstudio-builder-2d80q9915-webstudio-is.vercel.app/ gives empty page

**Review by istarkov — CHANGES_REQUESTED** [review]
plz fix redirect
```if (buildParams === undefined) {
    throw redirect(dashboardPath());
  }
```

**Comment by istarkov** [comment]
> is there anything we also need to do in saas for this?

no, everything should be ok

**Review by istarkov — APPROVED** [review]

**Pull request merged**